File: trunk/quests/nro/a_Dervish_Cutthroat.pl
sub EVENT_COMBAT {
	my $NPCRace = $npc->GetRace();
	if($combat_state == 1){
		if($NPCRace==54){
			quest::say("Orc stomp, orc kill - orc weapons, your blood will spill.");
		}
		else {
			quest::say("Time to die $name!");
		}
	}
}

sub EVENT_DEATH_COMPLETE {
		if($NPCRace!=54){
			quest::say("My comrades will avenge my death.");
		}
}

File: trunk/quests/nro/a_Dervish_Cutthroat.lua
function event_combat(e)
	if(e.joined == true) then
		if(e.self:GetRace() == 54) then
			e.self:Say("Orc stomp, orc kill - orc weapons, your blood will spill.");
		else
			e.self:Say(string.format("Time to die %s!",e.other:GetCleanName()));
		end
	end
end

function event_death_complete(e)
	if(e.self:GetRace() == 54) then
		e.self:Say("My comrades will avenge my death.");
	end
end
